events/wc_undead_events.txt

```diff
@@ -56,29 +56,29 @@ province_event = {
 				}
```

Comment:
You should block turning normal undead for Throsic, so we see 3 undead options and the option to leave the event chain (on the first screenshot, there's no option to leave like on the second one).
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/79164713-aa3e7900-7df2-11ea-80c2-aa5043b702a5.png)
![image](https://user-images.githubusercontent.com/46384992/79164847-f4275f00-7df2-11ea-8a7a-7947d446754d.png)

</details>

Reply:
@Velius94 it's not resolved.